multichain-testing/test/fast-usdc/fast-usdc.test.ts

```diff
@@ -107,23 +110,64 @@ test.after(async t => {
   deleteTestKeys(accounts);
 });
 
+type VStorageClient = Awaited<ReturnType<typeof commonSetup>>['vstorageClient'];
+const agoricNamesQ = (vsc: VStorageClient) =>
+  harden({
+    brands: <K extends AssetKind>(_assetKind: K) =>
+      vsc
+        .queryData('published.agoricNames.brand')
+        .then(pairs => fromEntries(pairs) as Record<string, Brand<K>>),
+  });
+const walletQ = (vsc: VStorageClient) => {
+  const self = harden({
+    current: (addr: string) =>
+      vsc.queryData(
+        `published.wallet.${addr}.current`,
+      ) as Promise<CurrentWalletRecord>,
+    findInvitationDetail: async (addr: string, description: string) => {
+      const { Invitation } = await agoricNamesQ(vsc).brands('set');
+      const current = await self.current(addr);
+      const { purses } = current;
+      const { value: details } = purses.find(p => p.brand === Invitation)!
+        .balance as Amount<'set', InvitationDetails>;
+      const detail = details.find(x => x.description === description);
+      return { current, detail };
+    },
+  });
+  return self;
+};
+
+const fastLPQ = (vsc: VStorageClient) =>
+  harden({
+    metrics: () =>
+      vsc.queryData(
+        `published.${contractName}.poolMetrics`,
```

Comment:
I don't get why this is parameterized for any contractName when it's bound to `fastLPQ`.

Reply:
copy/paste left-over.

if I change anything else, I'll remove the parameter. not worth another round of ci on its own, I suggest